src/webgpu/shader/execution/robust_access_vertex.spec.ts

```diff
@@ -573,13 +548,11 @@ g.test('vertex_buffer_access')
           if (p.indirect) {
             yield* ['indexCount', 'instanceCount', 'firstIndex', 'firstInstance'] as const;
           }
-        } else {
-          if (p.indirect) {
-            yield* ['vertexCount', 'instanceCount', 'firstVertex', 'firstInstance'] as const;
-          }
+        } else if (p.indirect) {
+          yield* ['vertexCount', 'instanceCount', 'firstVertex', 'firstInstance'] as const;
         }
       })
-      .combine('type', Object.keys(typeInfoMap))
+      .combine('type', Object.keys(typeInfoMap) as GPUVertexFormat[])
```

Comment:
@kainino0x is there a better way to do this in TS? Otherwise looks good to me as is.